README.md

```diff
@@ -84,6 +84,11 @@ grunt.initConfig({
 });
 ```
 
+You can also inspect [`jsx` files used in React.js](http://facebook.github.io/react/docs/jsx-in-depth.html),
+simply by using that suffix in the `src` list.
+The command line version of `jsinspect` uses a specific flag for this,
+[namely `--jsx`](https://github.com/danielstjules/jsinspect#usage).
+
```

Comment:
Only specifying the files is needed.
The command line tool uses a flag for achieving the same behaviour